data.sql

```diff
@@ -1,5 +1,9 @@
 /* Populate database with sample data. */
 
-INSERT INTO animals (name) VALUES ('Luna');
-INSERT INTO animals (name) VALUES ('Daisy');
-INSERT INTO animals (name) VALUES ('Charlie');
+INSERT INTO animals (name, date_of_birth, weight_kg, neutered, escape_attempts) VALUES ('Agumon', date '02-03-2020', '10.23', 'true', '0');
+INSERT INTO animals (name, date_of_birth, weight_kg, neutered, escape_attempts) VALUES ('Gabumon', date '11-15-2018', '8', 'true', '2');
+INSERT INTO animals (name, date_of_birth, weight_kg, neutered, escape_attempts) VALUES ('Pikachu', date '01-17-2021', '15.04', 'false', '1');
+INSERT INTO animals (name, date_of_birth, weight_kg, neutered, escape_attempts) VALUES ('Devimon', date '05-12-2017', '11', 'true', '5');
+
+
+
```

Comment:
- [ ] You are not using the latest verison of microverse readme template 🤔 Kindly use the latest version 👍 You can get it from this [link](https://github.com/microverseinc/readme-template/blob/master/README.md) 😊

Reply:
## Hello @HAMMAS-SALEEM   👋🏼 
 Thanks for taking out time to review this PR. The required and optional changes have been duly noted and necessary changes will be implemented.